haskell/private/expansions.bzl

```diff
@@ -0,0 +1,20 @@
+def expand_make_variables(name, ctx, strings, extra_label_attrs):
+    # All labels in all attributes should be location-expandable.
+    label_attrs = [
+        ctx.attr.deps,
```

Comment:
Why assume that all rules have a `deps` attribute instead of taking `label_attrs` as a parameter to `expand_make_variables`?

Reply:
I was under the impression that these particular attributes are common to all build rules (https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes), so it felt redundant/error-prone to pass them all the time.

Can change it back if you'd rather this to be explicit!

Reply:
Fair enough, thanks for explaining. I have no strong preference either way.